src/Atmos/Model/bc_energy.jl

```diff
@@ -114,3 +114,67 @@ function atmos_energy_normal_boundary_flux_second_order!(
     # we want to prescribe the inward flux
     fluxᵀn.ρe -= bc_energy.fn(state⁻, aux⁻, t)
 end
+
+"""
+    BulkFormulaEnergy(fn) :: EnergyBC
+
+Calculate the net inward energy flux across the boundary by `fn`, a function
+with signature `fn(state, aux, t)`, returning the flux (in W/m^2).
+"""
+struct BulkFormulaEnergy{FN} <: EnergyBC
+    fn::FN
+end
+function atmos_energy_boundary_state!(
+    nf,
+    bc_energy::BulkFormulaEnergy,
+    atmos,
+    state⁺,
+    aux⁺,
+    n,
+    state⁻,
+    aux⁻,
+    bctype,
+    t,
+    args...,
+) 
+end
+function atmos_energy_normal_boundary_flux_second_order!(
+    nf,
+    bc_energy::BulkFormulaEnergy,
+    atmos,
+    fluxᵀn,
+    n⁻,
+    state⁻,
+    diff⁻,
+    hyperdiff⁻,
+    aux⁻,
+    state⁺,
+    diff⁺,
+    hyperdiff⁺,
+    aux⁺,
+    bctype,
+    t,
+    state1⁻,
+    diff1⁻,
+    aux1⁻,
+)
+
+    u1⁻ = state1⁻.ρu / state1⁻.ρ
+    Pu1⁻ = u1⁻ - dot(u1⁻, n⁻) .* SVector(n⁻)
```

Comment:
Didn't we have a function (`tangential_velocity`?) for this somewhere? (I know it came up in the Held-Suarez setup.) If we don't have it, we should probably add it, since we'll need in several places.

Reply:
And could we have some more descriptive variable names instead of `state1⁻` and `state⁻` etc.? (e.g., `...int` and `...sfc`?)

Reply:
> And could we have some more descriptive variable names instead of `state1⁻` and `state⁻` etc.? (e.g., `...int` and `...sfc`?)

I agree with this! I am inheriting the code from bc_momentum.jl. As someone who looks at bc for the first time it's really hard to understand what these variables mean.